### DISCUSSION TITLE
How one can change the institutuion related to domain name?

### DISCUSSION THREAD

**Discussion by HemboPagi** [community/community#64198] [GitHub Education]
### Hi everyone,

Hello

I discovered that currently tlu.ee domain is linked with the Tallinn University Haapsalu College which is not correct. I want to sign up as Tallinn University but i can not change the institution. Also, currently [signup does not work](https://github.com/orgs/community/discussions/49378).

All the best
Hembo

**Comment by tooltham** [comment]
So that you know, about "currently signup does not work," I faced a similar problem and resolved it using Google Chrome. To fix the issue, I granted permission to Github to access my location and enabled location access in my browser. This solution worked like magic.

**Comment by HemboPagi** [comment]
Finally i got it the signup to work with Google Chrome. I did not work with Safari an Brave.
Still the issue with the institution reminds
